api_pagination/__init__.py

```diff
@@ -0,0 +1,95 @@
+# Load in our dependencies
+from math import ceil
+
+
+# Define our class
+class Paginator(object):
+    def __init__(self, total, items_per_page):
+        """
+        Constructor for pagination class
+
+        :param int total: Count of items we are paginating
+        :param int items_per_page: How many items to place on each page
+        """
+        # Save our information for later
+        self.total = total
+        self.items_per_page = items_per_page
+
+    def get_page_count(self):
+        """
+        Calculates the amount of pages that exist
+
+        Note: If your data has 0 items per page, you will receive a `ZeroDivisionError`
+
+        :rtype: int
+        :returns: Overall amount of pages (e.g. total=50, items_per_page=10 -> 5 pages)
+        """
+        # DEV: We must perform float division since we want to always return partial pages
+        #   (e.g. total=5 but items=1, we should still have 1 page
```

Comment:
this comment is backwards, we want `total=1` and `items=5` so the division will be `ceil(1/5)` to yield 1 page.

also missing the closing paren.